dxr/static_unhashed/js/dxr.js

```diff
@@ -245,6 +245,130 @@ $(function() {
     }
 
     /**
+     * Visit the result lines in result and remove duplicated lines and redundant context buttons.
+     */
+    function cleanupResults(result) {
+        // Construct {line: {hasContext, hasResult}}
+        var lineMap = {};
+        // Visit the lines and remove duplicates.
+        result.children('.result_line').each(function() {
+            var line = parseInt($(this).data("line"));
```

Comment:
You should save `$(this)` like `var $this = $(this)` if you're going to re-use it.